openedx/core/djangoapps/user_api/accounts/tests/test_permissions.py

```diff
@@ -67,3 +76,37 @@ def test_api_permission_user_without_permission(self):
         self.request.user = UserFactory()
         result = CanRetireUser().has_permission(self.request, None)
         assert not result
+
+
+class CanCancelUserRetirementTest(TestCase):
+    """ Tests for cancel user retirement API permissions """
+
+    def setUp(self):
+        super().setUp()
+        self.request = RequestFactory().get('/test/url')
```

Comment:
curious: Why is a get request made?

Reply:
Just to instantiate a test request to check for permissions.